src/ride/vehicle.c

```diff
@@ -7103,7 +7104,6 @@ loc_6DBB08:;
 	}
 
 loc_6DBB7E:;
-	int direction;
```

Comment:
This isn't quite correct the direction should be outputDirection when evaluated through the 6DBB4F branch.